cloud/vmware/vca_vapp.py

```diff
@@ -0,0 +1,747 @@
+#!/usr/bin/python
+
+# Copyright (c) 2015 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+
+DOCUMENTATION = '''
+---
+module: vca_vapp
+short_description: create, terminate, start or stop a vm in vca
+description:
+  - Creates or terminates vca vms.
+version_added: "2.0"
+options:
+    username:
+      version_added: "2.0"
```

Comment:
no need for version_added on options for a new module, they are assumed to have been added when modules was added